### PR TITLE
Modify hereditary hemophagocytic lymphohistiocytosis

### LINKED ISSUE #7437: hereditary hemophagocytic lymphohistiocytosis
hereditary hemophagocytic lymphohistiocytosis MONDO:0015541

I believe this should not be linked to the group, (Primary hemophagocytic lymphohistiocytosis), only to the disorder: Familial hemophagocytic lymphohistiocytosis https://www.orpha.net/en/disease/detail/540 (the synonyms should also be fixed)

Thank you,

Gioconda Alyea

### PR DESCRIPTION
**Pull request by twhetzel**
closes #7437

@sabrinatoro can you check if I've made the correct changes based on the [last comment ](https://github.com/monarch-initiative/mondo/issues/7437#issuecomment-2025921543)in the ticket?

### CODE REVIEW

**Comment by twhetzel** [comment]
@sabrinatoro I don't see your comments. Did you finish the review?

**Comment by sabrinatoro** [comment]
@twhetzel I didn't comment about the synonyms. But for the synonyms with the orphanet to be exclude, please keep them (because we might have reviewed them previously and OKed them and we didn't keep track of it).

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -227169,6 +227169,7 @@ is_a: MONDO:0016537 {source="DC-OMIM:308240", source="DOID:0060705", source="MON
 is_a: MONDO:0021094 {source="NCIT:C61246"} ! immunodeficiency disease
 intersection_of: MONDO:0016537 ! lymphoproliferative syndrome
 intersection_of: has_characteristic HP:0001417 ! X-linked inheritance
+relationship: excluded_subClassOf MONDO:0015541 {source="Orpha:158038"} ! hereditary hemophagocytic lymphohistiocytosis
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:227172]: done

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -343030,23 +343031,16 @@ def: "An instance of hemophagocytic lymphohistiocytosis that is caused by an inh
 subset: disease_grouping
 subset: gard_rare {source="GARD:20025", source="GARD:6589"}
 subset: nord_rare {source="MONDO:NORD"}
-subset: ordo_group_of_disorders {source="Orphanet:158038"}
-subset: orphanet_rare {source="Orphanet:158038", source="Orphanet:540"}
+subset: orphanet_rare {source="Orphanet:540"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:343034]: done

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -343030,23 +343031,16 @@ def: "An instance of hemophagocytic lymphohistiocytosis that is caused by an inh
...
-xref: GARD:20025 {source="Orphanet:158038"}
 xref: GARD:6589 {source="Orphanet:540"}
 xref: ICD9:238.79 {source="MONDO:relatedTo", source="MONDO:i2s"}
-xref: MedDRA:10070904 {source="Orphanet:158038", source="Orphanet:158038/e"}
 xref: OMIMPS:267700 {source="MONDO:equivalentTo", source="https://orcid.org/0000-0002-6601-2165"}
-xref: Orphanet:158038 {source="MONDO:equivalentTo"}
 xref: Orphanet:540 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
 xref: SCTID:398250003 {source="MONDO:equivalentTo"}
-xref: UMLS:C0272199 {source="Orphanet:158038", source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="Orphanet:158038/e"}
 is_a: MONDO:0003778 {source="https://orcid.org/0000-0001-5208-3432"} ! inborn error of immunity
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:343044]: @sabrinatoro to add back `xref: Orphanet:158038 `, what should the source be?

[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:343044]: no source

[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:343044]: the others are updated, but have not had confirmation of these steps from @nicolevasilevsky yet

[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:343044]: sounds good! Thanks to you both!

**Review by nicolevasilevsky — APPROVED** [review]

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel 
The following should be kept (they still look as removed), because we might have reviewed and agreed to these synonyms, but we did not indicate it. In addition, we should keep that GARD, even if the orphanet is excluded.
Screenshot 2024-05-03 at 12 04 06 PM

Also, please make sure to resolve the QC fail. 
Thanks

**Comment by twhetzel** [comment]
Addressed CR (change request)

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
just a few things to change

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -227196,11 +227196,11 @@ xref: Orphanet:2442 {source="MONDO:equivalentTo", source="GARD:0007906", source=
 xref: SCTID:77121009 {source="MONDO:equivalentTo"}
 xref: UMLS:C0549463 {source="Orphanet:2442", source="MONDO:equivalentTo", source="Orphanet:2442/e", source="OMIM:308240", source="NCIT:C61246"}
 is_a: MONDO:0000425 {source="MONDO:Redundant"} ! X-linked disease
-is_a: MONDO:0015541 {source="Orphanet:2442"} ! hereditary hemophagocytic lymphohistiocytosis
 is_a: MONDO:0016537 {source="DC-OMIM:308240", source="DOID:0060705", source="MONDO:Redundant", source="Orphanet:2442"} ! lymphoproliferative syndrome
 is_a: MONDO:0021094 {source="NCIT:C61246"} ! immunodeficiency disease
 intersection_of: MONDO:0016537 ! lymphoproliferative syndrome
 intersection_of: has_characteristic HP:0001417 ! X-linked inheritance
+relationship: excluded_subClassOf MONDO:0015541 {source="Orphanet:158038", source="https://orcid.org/0000-0001-9310-0163"} ! hereditary hemophagocytic lymphohistiocytosis
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:227203]: done

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -343112,12 +343112,12 @@ synonym: "familial hemophagocytic lymphohistiocytosis" EXACT [https://orcid.org/
 synonym: "genetic hemophagocytic lymphohistiocytosis" EXACT [Orphanet:158038]
 synonym: "genetic hemophagocytic syndrome" EXACT [MONDO:patterns/genetic]
 synonym: "primary hemophagocytic lymphohistiocytosis" EXACT [Orphanet:158038]
-xref: GARD:20025 {source="Orphanet:158038"}
+xref: GARD:20025
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:343115]: done

File: src/ontology/mondo-edit.obo (hunk 4 above)
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:227203]: How is `Orpha:158038` excluded in Mondo as a grouping term that is not in scope in Mondo from [this comment](https://github.com/monarch-initiative/mondo/issues/7437#issuecomment-2025921543)?

[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:227203]: You need to add this x-ref in the "excluded" file for Orphanet. I will add a note in the issue.

**Review by sabrinatoro — APPROVED** [review]
Everything looks good. Approving and merging.

**Pull request merged**